src/exec_auth.ts

```diff
@@ -4,12 +4,25 @@ import { Authenticator } from './auth';
 import { User } from './config_types';
 
 export class ExecAuth implements Authenticator {
+    private tokenCache: any = {};
```

Comment:
Could type be `{ [key: string]: SomeTokenType }`?

Reply:
done.